src/main/java/Duke.java

```diff
@@ -1,10 +1,136 @@
+                    }
+                } else if (line.contains("done")) {
+/**
+ * //refers to list index is done and mark it
+ */                 int i = Integer.parseInt(line.substring(5)) - 1;
+                    Task currentTask = tasks.taskList.get(i);
+                    currentTask.markDone();
+                    System.out.println("   Nice! I've marked this task as done:\n " + currentTask.toString());
+                } else if (line.contains("delete")) {
+                    int delete = Integer.parseInt(line.substring(7)) - 1;
+                    Task currentTask = tasks.taskList.get(delete);
+                    tasks.removeTask(delete);
+                    pointer++;
+                } else if (line.contains("todo")) {
+                    Task todo = new ToDo(line.substring(5));
+                    tasks.addTask(todo);
+                    System.out.println("Got it. I've added this task:");
+                    System.out.println(tasks.get(tasks.taskList.size() - 1).toString());
+                    System.out.println("Now you have " + (tasks.taskList.size()) + " tasks in the list.");
+                    pointer++;
+                } else if (line.contains("event")) {
+                    int separator = line.indexOf('/');
+                    String dateAndTime = line.substring(separator + 4);
+                    DateTimeFormatter format = DateTimeFormatter.ofPattern("dd/MM/yyyy HHmm");
+                    LocalDateTime dateTime = LocalDateTime.parse(dateAndTime, format);
+                    Task event = new Event(line.substring(6, separator - 1), dateTime);
+                    tasks.addTask(event);
+                    System.out.println("Got it. I've added this task:");
+                    System.out.println(tasks.get(tasks.taskList.size() - 1).toString());
+                    System.out.println("Now you have " + (tasks.taskList.size()) + " tasks in the list.");
+                    pointer++;
+                } else if (line.contains("reminder")) {
+                    int separator = line.indexOf('/');
+                    String dateAndTime = line.substring(separator + 4);
+                    DateTimeFormatter format = DateTimeFormatter.ofPattern("dd/MM/yyyy HHmm");
+                    LocalDateTime dateTime = LocalDateTime.parse(dateAndTime, format);
+                    Task reminder = new Reminder(line.substring(9, separator - 1), dateTime);
+                    tasks.addTask(reminder);
+                    System.out.println("Got it. I've added this reminder:");
+                    System.out.println(tasks.get(tasks.taskList.size() - 1).toString());
+                    System.out.println("Now you have " + (tasks.taskList.size()) + " reminder in the list.");
+                    pointer++;
+                } else if (line.contains("deadline")) {
+/**
+ * Deadline class syntax where user inputs in format -  Deadline /by 12/12/2020 1230, return in localDateTime format
+ */
```

Comment:
Javadoc style comments are only to be used to document classes, methods, and variables of a class (not local variables inside a method).